### PR TITLE
Fix in SafeCredentialHandle when using default credentials

### PR DESCRIPTION
**Pull request by kapilash**
This PR fixes the Null Reference Exception when default credentials are used.

### CODE REVIEW

**Comment by kapilash** [comment]
cc @vijaykota @stephentoub

File: src/Common/src/Microsoft/Win32/SafeHandles/GssSafeHandles.cs [hunk 1]
```diff
@@ -86,32 +86,33 @@ internal class SafeGssCredHandle : SafeHandle
         /// </summary>
         public static SafeGssCredHandle Create(string username, string password, string domain)
         {
-            SafeGssCredHandle retHandle = null;
```
Comment (vijaykota) [inline src/Common/src/Microsoft/Win32/SafeHandles/GssSafeHandles.cs]: The diffs would have been easier to read if you simply change the init of retHandle :) thusly:
````
SafeGssCredHandle retHandle = new SafeGssCredHandle; // invalid handle
````

**Comment by vijaykota** [comment]
LGTM

**Comment by iamjasonp** [comment]
Thanks @kapilash!

File: src/Common/src/Microsoft/Win32/SafeHandles/GssSafeHandles.cs (hunk 1 above)
Comment (stephentoub) [inline src/Common/src/Microsoft/Win32/SafeHandles/GssSafeHandles.cs]: Just FYI, you can tag "w=1" on to the query string to see a diff with whitespace-differences ignored, e.g. https://github.com/dotnet/corefx/pull/6111/files?w=1

**Comment by stephentoub** [comment]
LGTM2

File: src/Common/src/Microsoft/Win32/SafeHandles/GssSafeHandles.cs (hunk 1 above)
Comment (vijaykota) [inline src/Common/src/Microsoft/Win32/SafeHandles/GssSafeHandles.cs]: > see a diff with whitespace-differences ignored

Neat trick, thanks!

**Comment by stephentoub** [comment]
@dotnet-bot test this please

**Comment by stephentoub** [comment]
Test Innerloop OSX Debug Build and Test please

**Pull request merged**